theano/sandbox/rng_mrg.py

```diff
@@ -887,11 +887,12 @@ def binomial(self, size, n=1, p=0.5,
             A Theano tensor of samples randomly drawn from a binomial distribution.
 
         """
-        if isinstance(n, (np.integer, integer_types)):
+        # TODO: use ABCs for these checks?
+        if not isinstance(p, theano.Variable):
```

Comment:
I think it should be `n` instead of `p` here, isn't it ?

Also, maybe add after that line something like:
```python
assert n == int(n), "n must be an integer"
```